helm/overrides/local.yaml

```diff
@@ -82,6 +84,7 @@ cronJobs:
       - "bash"
     args:
       - "-c"
-      - "pg_dump --no-owner --no-privileges | gzip > test.gz"
+      # supply the desired s3 bucket you wish to test with
+      - "pg_dump --no-owner --no-privileges | gzip > test.gz | aws s3 cp test.gz <insert-bucket-name>"
```

Comment:
Should we use an envvar instead of an actual placeholder? Seems like it serves the same purpose but is also more functional

Reply:
so something like `pg_dump --no-owner --no-privileges | gzip > test.gz | aws s3 cp test.gz s3://$S3_BUCKET_ROUTE/`? I think hat works. we would just need to mandate that that env var is set so the test can be actually run.